content/applications/hr/payroll.rst

```diff
@@ -0,0 +1,10 @@
+:nosearch:
+
+=======
+Payroll
+=======
+
```

Comment:
Good idea- I think I didn't add anything since I was going to have that overview doc- but I agree, we need something in the beginning while this is a one-off doc for now, till I write more.  I added a brief overview, and then mentioned the 3 items- I think this is a better way to open the doc. Thanks for the suggestion!

Reply:
See my suggestion on the intro of payslips.rst, I was referring to adding the intro here 🙂

```suggestion
In Odoo, the *Payroll* application handles the processing of payslips and the distribution of
payments, in addition to managing work entries, reports, employee information, and contracts.
```